ui/pages/home/home.component.js

```diff
@@ -700,6 +700,7 @@ export default class Home extends PureComponent {
                 data-testid="home__asset-tab"
                 name={this.context.t('assets')}
                 tabKey="assets"
+                boxProps={{ marginBottom: 2 }}
```

Comment:
Hmmm, I hate going around on this, but what if we skip the new `boxProps` and simply surround `<AssetList>` with a `<Box marginTop={2}>`?

Reply:
AssetList is a non-multichain component and we only needs to update the MultichainTokenListItem. I still think going with the className approach and adding css could be a better fix since MultichainTokenListItem accept className as a prop and we can update the css for native token. wdyt?

Reply:
Ah! My bad. Updated!! Thanks @darkwing